features/developer_portal/admin/account/personal_details_sso.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Dev Portal Buyer Personal Details
```

Comment:
It has the same name as the other feature. Maybe:
```suggestion
Feature: Dev Portal Buyer Personal Details SSO
```

Also add some clarification in the feature description, like.. what SSO means in this context hehe.